src/huggingface_hub/hf_api.py

```diff
@@ -140,6 +140,71 @@
 R = TypeVar("R")  # Return type
 CollectionItemType_T = Literal["model", "dataset", "space", "paper"]
 
+ExpandModelProperty_T = Literal[
+    "author",
+    "cardData",
+    "config",
+    "createdAt",
+    "disabled",
+    "downloads",
+    "downloadsAllTime",
+    "gated",
+    "gitalyUid",
```

Comment:
hmm this is internal only i don't think any user will ever need it

Reply:
i'd suggest removing that one

Reply:
opened https://github.com/huggingface/huggingface_hub/pull/2395